### PR TITLE
Renamed Rug plot dialog to Heat Map

### PR DESCRIPTION
**Pull request by EstherNjeriLiberatta**
This is opened because #6780 had issues and has been closed. 

@shadrackkibet this had been reviewed approved by @rdstern so please review it too.

### CODE REVIEW

**Comment by EstherNjeriLiberatta** [comment]
@africanmathsinitiative/developers @shadrackkibet This is ready for review.

File: instat/dlgRugPlot.vb [hunk 1]
```diff
@@ -343,4 +343,8 @@ Public Class dlgRugPlot
         End If
 
     End Sub
-End Class
+
+    Private Sub AllControlsContentsChanged(ucrChangedControl As ucrCore) Handles ucrVariableAsFactorForHeatMap.ControlContentsChanged, ucrSaveGraph.ControlContentsChanged, ucrReceiverX.ControlContentsChanged, ucrReceiverFill.ControlContentsChanged, ucrChkAddLabels.ControlContentsChanged
+
+    End Sub
```
Comment (shadrackkibet) [inline instat/dlgRugPlot.vb:349]: Do you need this?

File: instat/dlgRugPlot.vb [hunk 2]
```diff
@@ -300,7 +300,7 @@ Public Class dlgRugPlot
         bResetSubdialog = False
     End Sub
 
-    Private Sub cmdHeatMapOptions_Click(sender As Object, e As EventArgs) Handles cmdHeatMapOptions.Click
+    Private Sub cmdHeatMapOptions_Click(sender As Object, e As EventArgs) Handles cmdTileOptions.Click
```
Comment (shadrackkibet) [inline instat/dlgRugPlot.vb:303]: Could you make this consistent?

**Review by shadrackkibet — CHANGES_REQUESTED** [review]
The name of the dialog still reads `dlgRugPlot` I think you should change it as well.

**Comment by EstherNjeriLiberatta** [comment]
Hello @shadrackkibet, this is ready for review.

**Review by shadrackkibet — APPROVED** [review]

**Review by rdstern — APPROVED** [review]

**Pull request merged**